includes/REST_API/Link_Controller.php

```diff
@@ -131,6 +131,11 @@ public function parse_link( $request ) {
 
 		$response = wp_safe_remote_get( $url, $args );
 
+		$redirect_location = $response['headers']['location'];
```

Comment:
This will cause a fatal error if `$response` is an error.

Reply:
I'll leave this to another PR